guides/v15-mdc-migration.md

```diff
@@ -0,0 +1,217 @@
+  infrastructure
+
+## What has changed?
+
+The new components have different internal DOM and CSS styles. However, the TypeScript APIs and
+component/directive selectors for the new components have been kept as close as possible to the old
+implementation. This makes it straightforward to migrate your app and get it running with the new
+components.
+
+Due to the new DOM and CSS, you will likely find that some styles in your app need to be adjusted,
+particularly if your CSS is overriding styles on internal elements on any of the migrated
+components.
+
+There are a few components with larger changes to their APIs that were necessary in order to
+integrate with MDC. These components include:
+* form-field
+* chips
+* slider
+* list
+
+See below for a [detailed list of changes](#Detailed List of Changes) to be aware of for these and
+other components.
+
+The old implementation of each new component is now deprecated, but still available under a "legacy"
+package. e.g. The old `mat-button` implementation can be used by importing the legacy button module.
+
+```ts
+import {MatLegacyButtonModule} from '@angular/material/legacy-button';
+```
+
+## How to Migrate
+
+Migrating an existing app to use MDC based components is a tool-assisted migration. The Angular
+Material team has built schematics to handle as much of the work as possible. However, some manual
+effort will be needed.
+
+You can reduce the amount of manual effort needed by ensuring that your app follows good practices
+before migrating.
+* Avoid overriding styles on internal Angular Material elements in your CSS as much as possible. If
+  you find yourself frequently overriding styles on internal elements, consider using a component
+  that is designed for more style customization, such as the ones available in the
+  [Angular CDK](/cdk).
+* Use [component harnesses](./using-component-harnesses) to interact with Angular Material
+  components in tests rather than probing internal elements, properties, or methods of the
+  component. Using component harnesses makes your tests easier to understand and more robust to
+  changes in Angular Material
+
+### 1. Update to Angular Material v15
+
+Angular Material now includes a schematic to help migrate pre-existing apps to use the new
+MDC-based components. To get started, upgrade your app to Angular Material 15.
+
+```shell
+ng update @angular/components^15
+```
+
+As part of this update, a schematic will run to automatically move your app to use the "legacy"
+packages containing the old component implementations. This provides a quick path to getting your
+app running on v15 with minimal manual changes.
+
+### 2. Run the MDC Migration Script
+
+After upgrading to v15, you can run the MDC Migration Script to switch from the legacy component
+implementations to the new MDC-based ones.
+
+```shell
+ng generate # TODO(wagnermaciel): Insert command here.
```

Comment:
Amy is currently working on getting ng generate working locally so I'll have a more concrete answer for these three q's you once she gets that set up

Reply:
Ah, nvm. Amy already answered 👍🏽